app_test.py

```diff
@@ -0,0 +1,5 @@
+import requests
+def test_get_locations_for_us_90210_check_city_equals_beverly_hills():
```

Comment:
Looks like pytest returns a fail if run on TravisCI with no tests (https://github.com/pytest-dev/pytest/issues/2239) so thank you for putting in a passing test.